### PR TITLE
MCLOUD-5929: Upgrade PHPUnit to version 8

### PR DESCRIPTION
**Pull request by G-Arvind**
### Description

[MCLOUD-5929](https://jira.corp.magento.com/browse/MCLOUD-5929):
Since PHPUnit 7 is already deprecated and PHPUnit 8 is stable, The PHPUnit should be upgraded to version 8

### Fixed Issues (if relevant)

1. magento/magento-cloud-docker#193: Upgrade PHPUnit to version 8

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages

### CODE REVIEW

**Comment by G-Arvind** [comment]
Hi,
The static-unit for PHP 7.1 is failing as the PHPUnit 8 doesn't support PHP 7.1
Screenshot from 2020-04-30 16-20-46

**Comment by shiftedreality** [comment]
@G-Arvind please exclude 7.1 checks

**Comment by G-Arvind** [comment]
> @G-Arvind please exclude 7.1 checks
HI @shiftedreality 
Excluded PHP 7.1 from Travis. (Build Succeeded)
Thanks

**Comment by shiftedreality** [comment]
Thanks @G-Arvind 

The changes overall look good, but since it affects FT, we'll need to discuss this internally if we're ok to drop PHP 7.1 support in the next minor release.

Putting on hold for now.

**Comment by YPyltiai** [comment]
Discussed on call today - good to drop PHP 7.1 for Docker 1.1.0.

File: composer.json [hunk 1]
```diff
@@ -21,9 +21,9 @@
     "require-dev": {
         "phpmd/phpmd": "@stable",
         "phpstan/phpstan": "^0.11",
-        "phpunit/phpunit": "^7.2",
+        "phpunit/phpunit": "^8.5",
```
Comment (shiftedreality) [inline composer.json:24]: Please bump PHP version to `^7.2` in `require` section

**Review by shiftedreality — CHANGES_REQUESTED** [review]

File: .travis.yml [hunk 2]
```diff
@@ -72,11 +64,7 @@ jobs:
       env:
         - TEST_SUITE=build-images
     - stage: test
-      php: '7.1'
```
Comment (shiftedreality) [inline .travis.yml:75]: Please also remove `PHP 7.1` images and `7.1` version from `\Magento\CloudDocker\Command\Image\GeneratePhp`

File: composer.json (hunk 1 above)
Comment (G-Arvind) [reply composer.json:24]: @shiftedreality Changed the PHP version to `^7.2`. Thanks.

File: .travis.yml (hunk 2 above)
Comment (G-Arvind) [reply .travis.yml:75]: @shiftedreality Removed the `PHP 7.1` images and `7.1` version from `magento-cloud-docker/src/Command/Image/GeneratePhp.php`.

Thanks.

**Review by shiftedreality — APPROVED** [review]

**Review by michaelcasey316 — APPROVED** [review]

**Comment by shiftedreality** [comment]
No functionality changed. QA is not needed.

**Pull request merged**